### PR TITLE
Lung cell types

### PR DESCRIPTION
**Pull request by mkeays**
Added four cell types based on the lung annotations from Lia and additional research (lung parenchyma resident eosinophil, blood vessel smooth muscle cell, lymphatic vessel smooth muscle cell, tracheobronchial smooth muscle cell).

### CODE REVIEW

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -22346,6 +22350,43 @@ AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0019003 "goblet cell of trac
...
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> obo:CL_0019015 <http://orcid.org/0000-0003-2034-601X>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> obo:CL_0019015 "2020-05-26T09:42:19Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:27548519") oboInOwl:hasExactSynonym obo:CL_0019015 "lung-resident eosinophil")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:27548519") oboInOwl:hasExactSynonym obo:CL_0019015 "rEos")
+AnnotationAssertion(rdfs:label obo:CL_0019015 "lung parenchyma resident eosinophil"@en)
+EquivalentClasses(obo:CL_0019015 ObjectIntersectionOf(obo:CL_0000774 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0008946)))
+
+# Class: obo:CL_0019017 (lymphatic vessel smooth muscle cell)
+
+AnnotationAssertion(obo:IAO_0000115 obo:CL_0019017 "A smooth muscle cell that is part of any lymphatic vessel")
```
Comment (nicolevasilevsky) [inline src/ontology/cl-edit.owl:22365]: could you please add a reference for the definition?

File: src/ontology/cl-edit.owl [hunk 2]
```diff
@@ -22346,6 +22350,43 @@ AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0019003 "goblet cell of trac
...
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> obo:CL_0019017 <http://orcid.org/0000-0003-2034-601X>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> obo:CL_0019017 "2020-07-21T12:29:51Z"^^xsd:dateTime)
+AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0019017 "smooth muscle cell of lymphatic vessel")
+AnnotationAssertion(rdfs:label obo:CL_0019017 "lymphatic vessel smooth muscle cell"@en)
+EquivalentClasses(obo:CL_0019017 ObjectIntersectionOf(obo:CL_0000192 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0001473)))
+
+# Class: obo:CL_0019018 (blood vessel smooth muscle cell)
+
+AnnotationAssertion(obo:IAO_0000115 obo:CL_0019018 "A smooth muscle cell that is part of any blood vessel")
```
Comment (nicolevasilevsky) [inline src/ontology/cl-edit.owl:22374]: Could you please add a reference for this def too (like a PMID)

File: src/ontology/cl-edit.owl [hunk 3]
```diff
@@ -22346,6 +22350,43 @@ AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0019003 "goblet cell of trac
...
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> obo:CL_0019018 <http://orcid.org/0000-0003-2034-601X>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> obo:CL_0019018 "2020-07-21T12:42:48Z"^^xsd:dateTime)
+AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0019018 "smooth muscle cell of blood vessel")
+AnnotationAssertion(rdfs:label obo:CL_0019018 "blood vessel smooth muscle cell"@en)
+EquivalentClasses(obo:CL_0019018 ObjectIntersectionOf(obo:CL_0000192 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0001981)))
+
+# Class: obo:CL_0019019 (tracheobronchial smooth muscle cell)
+
+AnnotationAssertion(obo:IAO_0000115 obo:CL_0019019 "A smooth muscle cell that is part of the tracheobronchial tree")
```
Comment (nicolevasilevsky) [inline src/ontology/cl-edit.owl:22383]: please add a reference here too, thanks!

**Comment by mkeays** [comment]
Sorry for the delay, have now added references for the above.

**Pull request merged**